tmt/utils.py

```diff
@@ -1304,19 +1304,16 @@ def option_to_key(option: str) -> str:
     return option.replace('-', '_')
 
 
-SerializableContainerDerivedType = TypeVar(
-    'SerializableContainerDerivedType',
-    bound='SerializableContainer')
-
-
 @dataclasses.dataclass
-class SerializableContainer:
-    """
-    A mixin class for objects that may be saved in files and restored later
-    """
+class KeyContainer:
```

Comment:
Might be. My `KeyContainer` was based on the concept of "keys" present in tmt sources. I don't mind changing it to `DataContainer`, it might be a bit too generic for my taste, but on the other hand, plenty of `data` objects got way more specific recently that I can live with `DataContainer`, especially when it's a base class for many of new additions.

Please, decide, I'll fix other issues in the meantime.

Reply:
I think it nicely works with `StepData`, let's go with the `DataContainer` then.

Reply:
Very well, fixed.